specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -1281,6 +1281,101 @@ STDAPI MddBootstrapInitialize(
 /// @warning Packages made available via MddBootstrapInitialize() and
 ///          the Dynamic Dependencies API should not be used after this call.
 STDAPI_(void) MddBootstrapShutdown() noexcept;
+
+// C++ friendly APIs
+#if defined(__cplusplus)
+#if defined(WINDOWSAPPSDK_RELEASE_MAJORMINOR) && defined(WINDOWSAPPSDK_RELEASE_VERSION_TAG_W) && defined(WINDOWSAPPSDK_RUNTIME_VERSION_UINT64)
```

Comment:
Can you add a sample use of the types? The test is instructive, but providing that canonical `WinMain` instance would help.

Reply:
Thanks for the reminder! I thought about that and got distracted